Include/internal/pycore_object.h

```diff
@@ -26,6 +25,42 @@ extern "C" {
         .ob_size = size, \
     }
 
+PyAPI_FUNC(void) _Py_NO_RETURN _Py_FatalRefcountErrorFunc(
+    const char *func,
+    const char *message);
+
+#define _Py_FatalRefcountError(message) _Py_FatalRefcountErrorFunc(__func__, message)
```

Comment:
I was trying to avoid a dependence between the `pycore_...` includes. The unicodeobject.c and tupleobject.c implementations both only used `pycore_pyerrors` for `_Py_FatalRefcountError()`, which doesn't really concern itself with the implementation details of exceptions.